devices/Sen5x/Entities/StringValue.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) 2017 The nanoFramework project contributors
+// See LICENSE file in the project root for full license information.
+using System;
+
+namespace Iot.Device.Sen5x.Entities
+{
+    /// <summary>
+    /// Represents a null-terminated string value read from the sensor.
+    /// </summary>
+    internal class StringValue : AbstractReadEntity
+    {
+        private int _maxLength;
```

Comment:
you should be able to have it readonly as well, doesn't seems you're changing the value anywhere (maybe in the underlying class but it doesn't seems so)